setup.cfg

```diff
@@ -0,0 +1,9 @@
+[codespell]
+ignore-words-list = miror
+
+[flake8]
+count = True
+max-complexity = 36
+max-line-length = 228
+select = E5,E9,F63,F7
```

Comment:
- I think you need to add `C9` here for the cyclomatic complexity check to work? (at least I did on my machine - try setting `max-complexity` to something lower to see what I mean)
- what do you think about including all `F` checks (and excluding the failing ones for now) like I did in https://github.com/RobotWebTools/rosbridge_suite/pull/605/files#diff-6951dbb399883798a226c1fb496fdb4183b1ab48865e75eddecf6ceb6cf46442R2-R4 ?

Reply:
I think that to F issues should be fixed because PEP8 is clear about wildcard imports being a bad idea.

Reply:
sorry I don't understand - can you rephrase?